src/fragments/lib/utilities/js/hub.mdx

```diff
@@ -73,7 +73,18 @@ The `event` field is recommended to be a small string without spaces such as `si
 
 ### Stop Listening
 
-Hub provides a way to stop listening for messages with `Hub.remove(channel: string | RegExp, listener: callback)`. This may be useful if you no longer need to receive messages in your application flow, as well as to avoid any memory leaks on low powered devices when you are sending large amounts of data through Hub on multiple channels.
+Hub provides a way to stop listening for messages with `Hub.remove(channel: string | RegExp, listener: callback)`:
+```javascript
+const callback = (data) => {
+  console.log('Listening for all messages: ', data.payload.data);
+}
+Hub.listen(/.*/, callback);
+
+/* later */
+Hub.remove(/.*/, callback);
+```
+
+This may be useful if you no longer need to receive messages in your application flow, as well as to avoid any memory leaks on low powered devices when you are sending large amounts of data through Hub on multiple channels.
```

Comment:
what is meant by channels here? I think that should be clarified a bit.

Reply:
@abdallahshaban557  The next paragraph talks about channels, do you think it would be better to swap the order of the "Stop Listening" and "Channels" sections?

Reply:
@helgabalashova I approve this change. Would you mind going ahead with it?